pkg/components/testframework/component.go

```diff
@@ -30,6 +30,12 @@ var TestFrameworkComponent = Component{
 			},
 
 			// TRT is owner of last resort for these
+			{
+				SIG:          "sig-arch",
```

Comment:
apiserver team wrote these.  seems reasonable (though not perfect) to assign to kube-apiserver

Reply:
I moved them